source/SemVerAnalyzer/Nuget/NugetClient.cs

```diff
@@ -21,7 +25,32 @@ public async Task<byte[]> GetAssemblyBytesFromPackage(string packageName, string
 			try
 			{
 				using var client = new HttpClient();
-				using var request = new HttpRequestMessage(HttpMethod.Get, Path.Join(_config.RepositoryUrl, packageName));
+				
+				using var feedRequest = new HttpRequestMessage(HttpMethod.Get, Path.Combine(_config.RepositoryUrl, "v3", "index.json"));
+				using var feedResponse = await client.SendAsync(feedRequest);
+				if (!feedResponse.IsSuccessStatusCode)
+				{
+					var content = await feedResponse.Content.ReadAsStringAsync();
+					comments.Add($"Error retrieving Nuget feed:\n{content}");
+					return null;
+				}
+
+				var feed = JsonSerializer.Deserialize<NugetPackageFeed>(await feedResponse.Content.ReadAsStringAsync());
```

Comment:
Please read the content into its own var.  It helps with debugging when the content isn't JSON (e.g. an HTML error page).